dapp_runner/_util.py

```diff
@@ -1,8 +1,28 @@
 from datetime import datetime, timezone
+import socket
+
 from yapapi import Golem, __version__ as yapapi_version
+
 from colors import yellow
 
 
+def get_free_port(range_start: int = 8080, range_end: int = 9090) -> int:
+    """Get the first available port on localhost within the specified range.
+
+    The range is inclusive on both sides (i.e. `range_end` will be included).
+    Raises `OverflowError` when no free port could be found.
+    """
+    for port in range(range_start, range_end + 1):
+        with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+            response = s.connect_ex(("localhost", port))
+            if response == 0:
+                return port
+
+    raise OverflowError(
```

Comment:
@zakaprov terribly sorry for one more late comment - but maybe we could consider raising a more generic `RuntimeError` here rather than, as far as I understand, an artithmetic/cpu-centric `OverflowError`? ...

Reply:
Done in 2bcca68